packages/nuxt/src/runtime/defineSdkConfig.template

```diff
@@ -12,6 +12,7 @@ type InjectedContext = {
    */
   getCookieHeader: () => Record<string, string>;
   getRequestHeaders: () => Record<string, string>;
+  defaults: ReturnType<typeof getDefaults>
```

Comment:
As this is an another quite static configuration data - make this as config object:

```suggestion
  config: {
    middlewareUrl: string;
    defaults: ReturnType<typeof getDefaults>
  }
```

And mark original one `middlewareUrl: string;` as deprecated

Reply:
Anyway to speed this up right now, we can skip deprecation, but only put it into the object ;)